common/class.Logger.php

```diff
@@ -182,19 +187,12 @@ public function register()
      * @param  int $errorLine
      * @return mixed
      */
-    public function log($level, $message, $tags, $errorFile = '', $errorLine = 0)
+    public function log($level, $message, $tags = [])
     {
         if ($this->enabled) {
             $this->disable();
             try {
                 if (defined('CONFIG_PATH')) {
-                    $tags = is_array($tags) ? $tags : [$tags];
-                    if (!empty($errorFile)) {
```

Comment:
This is a BC here. Are you sure that nothing is checking for these keys anywhere? (`file`, `line`)